app/views/admin/configs/index.haml

```diff
@@ -21,6 +21,12 @@
           = f.input :window_access_requires_release, label: 'Block access to the window if a client doesn\'t have a release?'
           = f.input :show_partial_ssn_in_window_search_results, label: 'Show partial SSNs in the window search results? (no means don\'t show anything)'
 
+      %section.o-section-card
+        %h3 System Cohorts
+        .c-card.c-card--flush.c-card--padded.flex-wrap
+          = f.input :enable_system_cohorts, label: 'Enable system cohorts?'
+          = f.input :currently_homeless_cohort, label: 'Currently Homeless', hint: 'The currently homeless cohort includes all open enrollment in ES, SO, TH, SH and no move-in date in a current PH project.'
```

Comment:
What is this doing?  Allowing you to turn on individual system cohorts?

Reply:
Yes -- seemed like once this exists people may want system cohorts, but need to be able to pick and choose which ones they get...